db/migrate/20170914094802_create_group_offers.rb

```diff
@@ -0,0 +1,34 @@
+class CreateGroupOffers < ActiveRecord::Migration[5.1]
+  def change
+    create_table :group_offers do |t|
+      t.string :title
+      t.string :offer_state
+      t.string :volunteer_state
+      t.integer :necessary_volunteers
+      t.string :volunteer_responsible
+      t.string :volunteer_responsible_name
```

Comment:
Why isn't that solved with relations?

Reply:
```ruby
has_one :responsible, class_name: 'User', foreign_key: 'responsible_id'
```

Reply:
or
```ruby
has_and_belongs_to_many :volunteer
```

Reply:
Because that is part of a [following story](https://trello.com/c/MGYQ5fwy/4-5-als-freiwilligenverantwortliche-r-m%C3%B6chte-ich-ein-oder-mehrere-freiwillige-mit-gruppenangeboten-matchen). Maybe I should remove this field because it will be reimplemented anyway in the following story.